articles/azure-fluid-relay/how-tos/rotate-fluid-relay-access-keys.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to rotate Azure Fluid Relay access keys
+title: Rotate Azure Fluid Relay access keys
+ms.date: 08/13/2024
+ms.service: azure-fluid
+ms.topic: reference
+---
+
+# How to rotate Fluid Relay Server access keys
+This article provides an overview of managing access keys (tenant keys) in Azure Fluid Relay. 
+
+## Primary / Secondary keys
+Customers use the access keys to sign access tokens used to access Azure Fluid Relay resources, and Azure Fluid Relay uses the keys to validate the tokens. 
+
+Two keys are associated with each AFR resource: a primary key and secondary key. The purpose of dual keys is to let you regenerate, or roll, keys, providing continuous access to your account and data. 
+
+## View your access keys
+
+### [Azure Portal](#tab/azure-portal)
+To see your access keys, on the left menu select **Settings**. Then, select **Access Keys**. Select the **Copy** button to copy the selected key.
+
+:::image type="content" source="../images/rotate-tenant-keys.png" alt-text="A screenshot of the access keys page on the Azure portal.":::
+
+### [PowerShell](#tab/azure-powershell)
+To retrieve your access keys with PowerShell, call the [Get-AzFluidRelayServerKey](/powershell/module/az.fluidrelay/get-azfluidrelayserverkey) command. 
+
+
+```azurepowershell
+Get-AzFluidRelayServerKey -FluidRelayServerName <Fluid Relay resource name> -ResourceGroup <resource group> -SubscriptionId <subscription id> 
+```
+
+### [Azure CLI](#tab/azure-cli)
+To retrieve your access keys with Azure CLI, use [az fluid-relay server list-key command](/cli/azure/fluid-relay/server?view=azure-cli-latest&preserve-view=true#az-fluid-relay-server-list-key).
+
+```azurecli
+az fluid-relay server list-key --resource-group <resource group> --server-name <Fluid Relay resource name> 
+```
+
+## Rotate your access keys
+Two access keys are assigned so that you can rotate your keys. Having two keys ensures that your application maintains access to Azure Fluid Relay throughout the process. You should rotate one of two keys at one time to avoid service interruptions. 
```

Comment:
```suggestion
Two access keys are assigned so that your Azure Relay Service does not have to be taken offline when you rotate a key. Having two keys ensures that your application maintains access to Azure Fluid Relay throughout the process. You should rotate one of two keys at one time to avoid service interruptions. 
```